File: add/codesnippet/CPP/m-system.windows.forms.c_72_1.cpp
   void button3_Click( Object^ /*sender*/, EventArgs^ /*e*/ )
   {
      try
      {
         BindingManagerBase^ myBindingManager1 = BindingContext[ myDataSet, "Customers" ];
         myBindingManager1->SuspendBinding();
      }
      catch ( Exception^ ex ) 
      {
         MessageBox::Show( ex->Source );
         MessageBox::Show( ex->Message );
      }
   }

File: add/codesnippet/VisualBasic/m-system.windows.forms.c_72_1.vb
   Private Sub button3_Click(sender As Object, e As EventArgs)
      Try
         Dim myBindingManager1 As BindingManagerBase = BindingContext(myDataSet, "Customers")
         myBindingManager1.SuspendBinding()
      Catch ex As Exception
         MessageBox.Show(ex.Source.ToString())
         MessageBox.Show(ex.Message.ToString())
      End Try
   End Sub 'button3_Click
   